src/models/two_nodes.js

```diff
@@ -133,8 +140,49 @@ export function two_nodes(
   };
 
   let joint_kwargs = Object.assign(defaults_kwargs, kwargs);
+
+  if (wme === undefined) {
+    wme = 0;
```

Comment:
keep the default values in the parameters here.
Maybe you did not understand why in the other function it was better to leave them to undefined
In the other function you needed two different default values depending in the unit system, so it is better to leave it to undefined and then assign them based in the unit system like
```javascript
some_var = units === "SI" ? si_default_value : ip_default_value
```
In here there is no point in doing it this way, it is way better to set in it in the parameters